python/interpret_community/widget/js/main.js

```diff
@@ -3,36 +3,39 @@ import ReactDOM from 'react-dom';
 
 import { ExplanationDashboard } from 'interpret-dashboard';
 
+const RenderDashboard = (divId, data) => {
+  let generatePrediction = (postData) => {
+    return fetch(data.predictionUrl, {method: "post", body: JSON.stringify(postData), headers: {
+      'Content-Type': 'application/json'
+    }}).then(resp => {
+      if (resp.status >= 200 && resp.status < 300) {
+        return resp.json()
+      }
+      return Promise.reject(new Error(resp.statusText))
+    }).then(json => {
+      if (json.error !== undefined) {
+        throw new Error(json.error)
+      }
+      return Promise.resolve(json.data)
+    })
+  }
 
-let generatePrediction = (postData) => {
-  return fetch(data.predictionUrl, {method: "post", body: JSON.stringify(postData), headers: {
-    'Content-Type': 'application/json'
-  }}).then(resp => {
-    if (resp.status >= 200 && resp.status < 300) {
-      return resp.json()
-    }
-    return Promise.reject(new Error(resp.statusText))
-  }).then(json => {
-    if (json.error !== undefined) {
-      throw new Error(json.error)
-    }
-    return Promise.resolve(json.data)
-  })
+  ReactDOM.render(<ExplanationDashboard
+      modelInformation={{modelClass: 'blackbox'}}
```

Comment:
how do we know this is always blackbox?

Reply:
its not used at this time, the explanations don't log their type, this is a place to support model type in the future